movo_robot/movo_bringup/launch/manipulation/movo_manipulation.launch

```diff
@@ -27,6 +30,10 @@
             <param name="jaco_dof" value="7dof" if="$(optenv MOVO_HAS_KINOVA_ARM_7DOF false)"/>
         </node>
 
+        <!-- Launches all action servers related to arm movement -->
+        <!-- TODO uncomment when launch can be timed after movo_ros:init_robot is launched -->
+        <!-- <include file="$(find movo_bringup)/launch/manipulation/moveit_action_servers.launch"/> -->
```

Comment:
Can we uncomment this is we take out the init sequence?

Reply:
i think we should delte the init sequence and move this to nightingale_manipulation.laumch